query/src/sql/planner/binder/bind_context.rs

```diff
@@ -135,8 +106,10 @@ impl BindContext {
         let mut result = vec![];
         if let Some(table) = table {
             for column_binding in self.columns.iter() {
-                if column_binding.table_name.eq(&table) && column_binding.column_name.eq(&column) {
-                    result.push(column_binding.clone());
+                if let Some(table_name) = &column_binding.table_name {
+                    if table_name.eq(&table) && column_binding.column_name.eq(&column) {
```

Comment:
How about using `==` directly?

BTW, in which situation `table_name` will `!= table`?

Reply:
In the case that there are multiple tables in current context. For example, `SELECT * FROM t, t1`.